keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,562 @@
+        )
+        x = layers.Conv2D(
+            filters=stem_filters,
+            kernel_size=3,
+            strides=2,
+            kernel_initializer=conv_kernel_initializer(),
+            padding="same",
+            use_bias=False,
+            name="stem_conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=BN_AXIS,
+            momentum=0.9,
+            name="stem_bn",
+        )(x)
+        x = layers.Activation(activation, name="stem_activation")(x)
+
+        # Build blocks
+        block_args = copy.deepcopy(block_args)
+        b = 0
+        blocks = float(sum(args["num_repeat"] for args in block_args))
+
+        pyramid_level_tracker = 1
+        for i, args in enumerate(block_args):
+            assert args["num_repeat"] > 0
+
+            # Update block input and output filters based on depth multiplier.
+            args["input_filters"] = round_filters(
+                filters=args["input_filters"],
+                width_coefficient=width_coefficient,
+                min_depth=min_depth,
+                depth_divisor=depth_divisor,
+            )
+            args["output_filters"] = round_filters(
+                filters=args["output_filters"],
+                width_coefficient=width_coefficient,
+                min_depth=min_depth,
+                depth_divisor=depth_divisor,
+            )
+
+            repeats = round_repeats(
+                repeats=args.pop("num_repeat"),
+                depth_coefficient=depth_coefficient,
+            )
+            for j in range(repeats):
+                # The first block needs to take care of stride and filter size
+                # increase.
+                if j > 0:
+                    args["strides"] = 1
+                    args["input_filters"] = args["output_filters"]
+
+                if args["strides"] != 1:
+                    pyramid_level_inputs[
+                        pyramid_level_tracker
+                    ] = x.node.layer.name
+                    pyramid_level_tracker += 1
+
+                block = get_block_conv(
+                    args,
+                    activation=activation,
+                    survival_probability=drop_connect_rate * b / blocks,
+                    name="block{}{}_".format(i + 1, chr(j + 97)),
+                )
+                x = block(x)
+                b += 1
+
+        # Build top
+        top_filters = round_filters(
+            filters=1280,
+            width_coefficient=width_coefficient,
+            min_depth=min_depth,
+            depth_divisor=depth_divisor,
+        )
+
+        x = layers.Conv2D(
+            filters=top_filters,
+            kernel_size=1,
+            strides=1,
+            kernel_initializer=conv_kernel_initializer(),
+            padding="same",
+            data_format="channels_last",
+            use_bias=False,
+            name="top_conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=BN_AXIS,
```

Comment:
I think in both cases in this file, `BN_AXIS` (3) is equivalent to the default axis of `-1`.
It's worth verifying but I think we can just get rid of this.

Reply:
Works!